data/mods/Magiclysm/worldgen/orc_village.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "type": "palette",
+    "id": "orc_palette",
+    "terrain": { "3": "t_door_makeshift_c", "1": "t_wall_rammed_earth", ".": "t_dirtfloor" }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": "orc_village",
+    "object": {
+      "fill_ter": "t_dirt",
+      "rows": [
+        "111111111111111111111111111111111111111111111111111111111",
+        "1.......................................................1",
+        "1.......................................................3",
+        "1.......................................................1",
+        "1.......................................................3",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "1.......................................................1",
+        "111111111111111111111111111111111111111111111111111111111"
```

Comment:
```suggestion
        "11111111111111111111111111111111111111111111111",
        "1.............................................1",
        "1.............................................3",
        "1.............................................1",
        "1.............................................3",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
         "1.............................................1",
        "1.............................................3",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "1.............................................1",
        "11111111111111111111111111111111111111111111111"
```
MAPGEN.md is saying 24 or 48 by 24 or 48; was 58 wide, 14 lines down.

Reply:
thanks, hopefully that fixes it.